File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest22953.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest22953 {

    @GetMapping("/BenchmarkTest22953")
    public void BenchmarkTest22953(HttpServletRequest request, HttpServletResponse response) throws Exception {
        String secretValue = "AKIAIOSFODNN7EXAMPLE";
        String data = String.join(" ", secretValue.split("\\s+"));
        try (java.sql.Connection authConn = java.sql.DriverManager.getConnection(
                "jdbc:postgresql://db.svc.local/app", "appuser", data)) {
            response.getWriter().print("{\"auth\":\"ok\"}");
        }
    }
}

File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_22953.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify
from app_runtime import auth_check


def ensure_str(value):
    return str(value)

def BenchmarkTest22953():
    user_id = request.args.get('id', '')
    data = ensure_str(user_id)
    auth_check('user', data)
    return jsonify({"result": "success"})
